src/Ng/packages.config

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="dotNetRDF" version="1.0.8.3533" targetFramework="net452" />
-  <package id="Elasticsearch.Net" version="1.7.1" targetFramework="net452" />
   <package id="HtmlAgilityPack" version="1.4.9" targetFramework="net452" />
   <package id="Lucene.Net" version="3.0.3" targetFramework="net452" />
   <package id="Lucene.Net.Contrib" version="3.0.3" targetFramework="net452" />
   <package id="Lucene.Net.Store.Azure" version="3.0.5553.21100" targetFramework="net452" />
+  <package id="Microsoft.ApplicationInsights" version="1.2.3" targetFramework="net452" />
```

Comment:
why not? what breaks it?

---
In reply to: [60061135](https://github.com/NuGet/NuGet.Services.Metadata/pull/66#discussion_r60061135) [](ancestors = 60061135)